internal/pkg/consensus/processor.go

```diff
@@ -645,3 +676,53 @@ func (p *DefaultProcessor) minerOwnerAddress(ctx context.Context, st state.Tree,
 	}
 	return address.NewFromBytes(ret[0])
 }
+
+func getOrCreateActor(ctx context.Context, st *state.CachedTree, store vm.StorageMap, addr address.Address, gt *vm.GasTracker) (*actor.Actor, address.Address, error) {
+	// resolve address before lookup
+	idAddr, found, err := ResolveAddress(ctx, addr, st, store, gt)
+	if err != nil {
+		return nil, address.Undef, err
+	}
+
+	if found {
+		act, err := st.GetActor(ctx, idAddr)
+		return act, idAddr, err
+	}
+
+	if addr.Protocol() != address.ID {
```

Comment:
This condition is always true given the current definition of `ResolveAddress`.  I think the whole if guard can go away

Reply:
Good catch.